FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_multitask/demo_config.h

```diff
@@ -111,8 +82,72 @@
 /**
  * @brief The port to use for the demo.
  *
- * #define democonfigMQTT_BROKER_PORT					( insert here. )
+ * In general, port 8883 is for secured MQTT connections, and port 1883 if not
+ * using TLS.
+ *
+ * @note Port 443 requires use of the ALPN TLS extension with the ALPN protocol
+ * name. When using port 8883, ALPN is not required.
+ *
+ * #define democonfigMQTT_BROKER_PORT    "...insert here..."
```

Comment:
If the customer did pick port 443, where is the ALPN entered?

Reply:
I believe it's `NetworkCredentials_t.pAlpnProtos`. I just copied this description from the mutual auth demo. Should I add additional information to this note?

Reply:
Ok so if the customer didn't enter it into this config file, they would have to enter it into the demo source?

Reply:
Yes, they'd have to change it in the demo

Reply:
I see ok.. it's a little weird to have to modify the demo. It's a bit different from other stuff we are doing, we have the customer just change a macro in this config file only. Maybe add some documentation about it?

Reply:
Added